### DISCUSSION TITLE
chartGPT联动

### DISCUSSION THREAD

**Discussion by SK-Luffa** [he3-app/he3-feedback#7] [Ideas]
我觉得如果类似于chartGTP这样的智能AI开放接口的话，那么he3如果能顺利接入，这对he3将是一次壮举

**Comment by zhenweiwang1990** [comment]
你是指在 He3 里面直接跟 ChatGPT 对话，还是说希望利用 AI 的能力增强部分 He3 的体验呢？

**Comment by SK-Luffa** [comment]
是后者，但我认为两者实际上并不冲突，为什么不能既给He3加一个buff，同时创建出类似于小爱或者chartGPT这类的一个助手呢

**Comment by susery** [comment]
我觉得chatGPT都已经能自己生成代码并且debug代码了，可以说chatGPT的数据集是He3的超集。如果能用chatGPT为什么还要用He3呢？如果He3想要发展下去，应该走的是肢解chatGPT或者其他ai的道路，提供更加具体的调试ai工具集，方便工程师、程序员方便的调试。

**Reply by hongchaodeng** [reply]
你好，感谢你的建议！

你说的内容长远来看我们是非常认可的。但短期内，ChatGPT 在一系列长尾工具、小场景上有效率低、速度慢、成本高的问题。甚至很多更长尾的问题，它是没法给出准确答案的。

另外，调试ai工具集，在 ChatGPT 不够成熟的这段时间里，用户也不会太多。与其去解决这个“未来”的问题，我们不如着眼于解决当前开发者的问题。

最后，任何软件工具都不是一成不变永久有用的。随着时间推移，我相信整个世界会随着 AI 的演进带来翻天覆地的变化。我们的目标是给开发者带来更多的便利。随着 AI 的发展 He3 也会持续地演进。如果 ChatGPT 真的能完全取代 He3，那我们也是非常开心的，因为到那时候作为开发者的我们也会发挥出更大的能量去干别的事情。

**Reply by susery** [reply]
你好，首先呢我没有否定He3的意思。
我觉得调试AI工具集本质上和现在He3在做的工具集是一致的，都是手动的、是工程师程序员手里的工具。至于等到趋势形成之后再做，莫非阁下觉得chatGPT会夭折？chatGPT这个官方声称的高自动化工具，且不论在其他领域会如何，但是在编程领域产生的影响是很重大的。解构构建AI过程的算法和数据结构，并提供更有效的工具，对当下对未来不也是意义重大？
我可以再提供一个建议，建议He3参考一下Kali Linux那样做一个类似的工具集发行版，做成应用也可、提供脚本亦可，分好常用功能和核心功能，核心功能就可以通过付费订阅的方式来作为盈利点，还有就是上面提到的算法、数据结构的有效工具，其实也是一个盈利点，唐突间望周知。
